sections/experiments/methods/deepwalk.tex

```diff
@@ -9,5 +9,5 @@ \subsubsection{DeepWalk}
 This is done using the language model SkipGram, and the result of this is a vector representation of each node in $d$-dimensional space.
 \\\\
 The next step is to then use the vector representations to generate recommendations.
-Here we opted to go for a k-nearest neighbor (kNN) approach where we find the 10 nearest user nodes to a user node $u$ using the vector representations.
-To then generate a rating prediction for a specific item $i$, we look at whether the 10 neighbors have rated item $i$ and use the average of their ratings of $i$ as the predicted rating for user $u$ on item $i$.
+Here we opted to go for a k-nearest neighbor (kNN) approach where we find the $k$ nearest user nodes to a user node $u$ using the vector representations.
```

Comment:
Skal der ikke stå 10 nu hvor det er det valg vi har taget i implementationen?

Reply:
Den her del handler vel om metoden generelt, ikke vores implementation af det?

Reply:
Men lige præcis knn delen er vel et implementations valg vi har taget for at kunne lave recommendations
Men så igen, det gør nok ikke så meget om der står k eller 10 :xdog: